website/docs/faqs/API/rotate-token.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How can I rotate my user API token?
+description: "Instructions on how to swap API token"
```

Comment:
Does this need to include the word "rotate"?

Reply:
d'oh! good callout!